### PR TITLE
Migrating DocumentAction generateTransferCertificate using Adapter

### PR DESCRIPTION
**Pull request by Tauqueer031**

### CODE REVIEW

File: src/main/java/org/ideoholic/curium/model/documents/service/DocumentService.java [hunk 1]
```diff
@@ -83,38 +86,39 @@ public ResultResponse transferCertificate(String branchid) {
...
+		String lastexam = DataUtil.emptyString(transferCertificateDto.getLastExam());
+		String failpass = DataUtil.emptyString(transferCertificateDto.getFailPass());
+		String firstsubject = DataUtil.emptyString(transferCertificateDto.getFirstSubject());
+		String secondsubject = DataUtil.emptyString(transferCertificateDto.getSecondSubject());
+		String thirdsubject = DataUtil.emptyString(transferCertificateDto.getThirdSubject());
+		String Fourthsubject = DataUtil.emptyString(transferCertificateDto.getFourthSubject());
+		String Fifthsubject = DataUtil.emptyString(transferCertificateDto.getFifthSubject());
+		String sixthsubject = DataUtil.emptyString(transferCertificateDto.getSixthSubject());
+		String pinfig = DataUtil.emptyString(transferCertificateDto.getPinFig());
+		String pinword = DataUtil.emptyString(transferCertificateDto.getPinFig());
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/documents/service/DocumentService.java]: `String pinword = DataUtil.emptyString(transferCertificateDto.getPinWord());`

File: src/main/java/org/ideoholic/curium/model/documents/service/DocumentService.java [hunk 2]
```diff
@@ -131,37 +135,33 @@ public String generateTransferCertificate() {
...
+				 transferCertificateResponseDto.setPinWord(pinword);	
+				 transferCertificateResponseDto.setDues(dues);
+				 transferCertificateResponseDto.setConcession(concession);
+				 transferCertificateResponseDto.setWorkingDays(workingdays);
+				 transferCertificateResponseDto.setPresent(present);
+				 transferCertificateResponseDto.setNcc(ncc);
+				 transferCertificateResponseDto.setGame(game);
+				 transferCertificateResponseDto.setConduct(conduct);
+				 transferCertificateResponseDto.setDateCert(datecert);
+				 transferCertificateResponseDto.setRemarks(Remarks);
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/documents/service/DocumentService.java:165]: Also need to add:
`transferCertificateResponseDto.setDateInWord(dateinword);`

File: src/main/java/org/ideoholic/curium/model/documents/action/DocumentActionAdapter.java [hunk 3]
```diff
@@ -123,4 +125,74 @@ public boolean transferCertificate() {
...
+		request.setAttribute("pinword", transferCertificateResponseDto.getPinWord());
+		request.setAttribute("dues", transferCertificateResponseDto.getDues());
+		request.setAttribute("concession",transferCertificateResponseDto.getConcession());
+		request.setAttribute("workingdays",transferCertificateResponseDto.getWorkingDays());
+		request.setAttribute("present", transferCertificateResponseDto.getPresent());
+		request.setAttribute("ncc",transferCertificateResponseDto.getNcc());
+		request.setAttribute("game", transferCertificateResponseDto.getGame());
+		request.setAttribute("conduct", transferCertificateResponseDto.getConduct());
+		request.setAttribute("datecert", transferCertificateResponseDto.getDateCert());
+		request.setAttribute("Remarks", transferCertificateResponseDto.getRemarks());
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/documents/action/DocumentActionAdapter.java:186]: Also need to add:
`request.setAttribute("dateinword", transferCertificateResponseDto.getDateInWord());`

**Review by reehanlalkhan — CHANGES_REQUESTED** [review]

File: src/main/java/org/ideoholic/curium/model/documents/service/DocumentService.java (hunk 1 above)
Comment (Tauqueer031) [reply src/main/java/org/ideoholic/curium/model/documents/service/DocumentService.java]: done

File: src/main/java/org/ideoholic/curium/model/documents/service/DocumentService.java (hunk 2 above)
Comment (Tauqueer031) [reply src/main/java/org/ideoholic/curium/model/documents/service/DocumentService.java:165]: done

File: src/main/java/org/ideoholic/curium/model/documents/action/DocumentActionAdapter.java (hunk 3 above)
Comment (Tauqueer031) [reply src/main/java/org/ideoholic/curium/model/documents/action/DocumentActionAdapter.java:186]: done

**Review by reehanlalkhan — APPROVED** [review]

**Pull request merged**